source/admin_manual/importing_mailboxes.rst

```diff
@@ -0,0 +1,47 @@
+.. _importing_mailboxes:
+
+===================
+Importing mailboxes
+===================
+
+For importing mails, dovecot has `doveadm import <https://wiki2.dovecot.org/Tools/Doveadm/Import>`_ command.
+
+For migrating mails, see :ref:`migrating_mailboxes`.
+
+If you want to import mail from an existing user on same system, or from some external location,
+and you are not doing a migration, but actual import, you can use the import command.
+
+The import command imports mails as is, and does not import UIDs or check if the mail is already there.
+
+Importing mail from other users
+-------------------------------
+
+To import mail from another user in the system, you can do
+
+::
+
+  doveadm import -U sourceuser -u destuser maildir: Imported ALL
+
+This will import all mails and folder structure from sourceuser to destuser, under folder ``Imported``.
+
+This will require that both sourceuser and destuser have same system UID.
+
+You can also use imap client to do the import.
+
+::
+
+  doveadm import -U sourceuser -u destuser imapc: Imported ALL
+
+This assumes you have configured imap client, see :ref:`migrating_mailboxes_imapc` for details.
+
+Importing mail from filesystem
+------------------------------
+
+You can also import mail from filesystem location.
+
+::
+
+  doveadm import -u destuser maildir:/opt/backup/destuser "" ALL
+
+This will restore all mails from backup into mailbox root, with folder structure.
```

Comment:
Could also say something like: The main difference here to the previous example is that the `-U` parameter is not given. This causes the source location to be opened as the destuser.